EIPS/eip-6059.md

```diff
@@ -450,6 +450,8 @@ This state change places the token in the pending array of the new parent token.
 
 The Nestable token standard has been made compatible with [ERC-721](./eip-721.md) in order to take advantage of the robust tooling available for implementations of ERC-721 and to ensure compatibility with existing ERC-721 infrastructure.
 
+The only incompatible part of the specification is the restriction to not use the token ID of 0.
```

Comment:
I'm fine with this change, but would probably recommend something more specific, like:

```suggestion
The only incompatibility with ERC-721 is that Nestable tokens cannot use a token ID of 0.
```